src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1549,9 +1549,11 @@ private static Document ParseDocumentPartial(JsonOperationContext context, ref T
             if (fields.Contain(DocumentFields.ChangeVector))
                 result.ChangeVector = TableValueToChangeVector(context, (int)DocumentsTable.ChangeVector, ref tvr);
 
+            if (fields.Contain(DocumentFields.Flags))
```

Comment:
This is probably a breaking change, we wanted to see which tests will fail to help us see where we need to fix

Reply:
I couldn't find anything else that needed to be changed becuse of this